### DISCUSSION TITLE
how to nuke school

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#286] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so i have the nuclear bomb but i dont have a B-29 "Superfortress" Enola Gay bomber to nuke the school

**Comment by 3kh0** [comment, marked as answer]
just keep it in your backpack

**Reply by Noob-ishh** [reply]
nah fam nuke em

**Reply by rwg100** [reply]
i stole the school phone off the wall and its in my backpack lolz!!

**Comment by Noob-ishh** [comment]
I'll lend you my bomber :)

**Comment by 3kh0** [comment]
for legal reasons thats a joke

**Comment by Sxri** [comment]
how come u havent launched it at 3kh0 yet, he removed my collaborator invite 🙄

**Reply by 3kh0** [reply]
your invite expired due to not claiming it, I added you back

**Comment by Pugknight126** [comment]
Can i get the zip bomb nuke to bomb my school 
![image](https://github.com/user-attachments/assets/63f78e87-7570-432e-822a-d2eeb6ad6bd9) saw this post and wanted it
